nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-{
-  "name": "Acadia National Park",
-  "params": [
-    "visibility": {
-      "summariesByYear": [
-        {
-          "year": 2000,
-          "conditionStatus": "Warrants Moderate Concern",
-          "conditionTrend": "Relatively Unchanged",
-          "conditionConfidence": "High Confidence",
-          "conditionSummaryCombined": "concern-unchanging-high",
-          "benchmarkChartYAxisLabel": "Haze Index, dv",
-          "benchmarkChartValue": 11.5,
-          "benchmarkChartConcern": 15,
-          "benchmarkChartAverage": 12,
-          "benchmarkChartGood": 9,
-          "benchmarkChartColor": "Yellow",
```

Comment:
the above 5 lines build the chart.
![screen shot 2018-11-09 at 12 59 57 pm](https://user-images.githubusercontent.com/3201135/48279708-671ef000-e41f-11e8-9a91-91cba53c381d.png)
Color can come through as a human-readable string ("yellow") and the front end can parse this to the various design requirements.

Reply:
Okay.